### DISCUSSION TITLE
Add entity to display SPOT price

### DISCUSSION THREAD

**Discussion by bodaw117** [ulic75/power-flow-card#119] [Ideas]
Would be fine to display spot price on Grid, same way as battery SOC fx.

**Comment by ulic75** [comment]
At this point I have no intention of adding feature that are outside mirroring what is displaying on official Energy Distribution card.

**Comment by ruteclrp** [comment]
That is a pity I think. In DK we have pricing varying by the hour  with significant difference between cheap hours and expensive hours - not least because of also variable grid charges. Consumer price may vary by up to 1 EUR between cheapest and most expensive hour within same day. the typical variance is in the order of 0.4 EUR.
Having a user defined sensor placed somewhere in the card with user defined title would be highly beneficial.

Please reconsider. (especially now that I have also suggested a fix for the flow rate issue you requested) :-)
